tests/bin/sgx_get_att_quote.c

```diff
@@ -22,5 +34,14 @@ int main(void) {
     if (size < 0)
         return 1;
 
+    /* check beginning of quote matches expected value */
+    for (i; i < 32; i++)
+    {
+        if (buf[i] != expected[i] && (buf[i])!= dummy[i])
```

Comment:
odd  parentheses

Reply:
lol.. I wondered why `cargo clippy` didn't complain :)

Reply:
You're right, that is weird... why did I write that? I fixed it. : )

Reply:
>> lol.. I wondered why cargo clippy didn't complain :)

> You're right, that is weird...

Not sure, if you recognized, that this is a C-file and not rust :)